tcms/core/tests/test_ajax.py

```diff
@@ -35,6 +44,48 @@ def test_empty_string_is_converted_to_bool(self):
         response = self.client.get(url)
         self.assertEqual(200, response.status_code)
 
+    def test_with_ulli_format(self):
+        """ When a request comes with info_type=categories, and format=ulli for given product_id,
+            we expect to receive all categories for that product, rendered as HTML unordered list
+        """
+
+        url = '%s?info_type=categories&product_id=%d&format=ulli' % (reverse('ajax-info'),
+                                                                     self.product.pk)
+
+        response = self.client.get(url)
+
+        self.assertContains(response, '<li>' + self.category_one.name + '</li>')
+        self.assertContains(response, '<li>' + self.category_two.name + '</li>')
```

Comment:
Add assertion for the `-- default --` value as well and make sure there are opening and closing ul tags as well.